scripts/validate_ci_tests.py

```
@@ -0,0 +1,332 @@
+import os
+import sys
+import subprocess
+import time
+from pathlib import Path
+
+
+def run_command(cmd, description, capture_output=True):
+    """Run a command and return the result."""
+    print(f"\n🔄 {description}")
+    print(f"   Command: {' '.join(cmd)}")
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)